### PR TITLE
RELATED: RAIL-4320 Documentation for the Web Components library

### PR DESCRIPTION
**Pull request by andriichumak**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1122/ : SUCCESS in 1m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/752/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1175/ : SUCCESS in 1m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/783/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1176/ : SUCCESS in 1m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/784/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1184/ : SUCCESS in 1m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/790/ : SUCCESS in 6s

File: docs/19_webcomponents_dashboard.md [hunk 1]
```diff
@@ -53,6 +50,6 @@ Events **do not bubble** and **are not cancelable**. Here is how you can subscri
 </script>
 ```
 
-[the full list of available localizations]:https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui/src/base/localization/Locale.ts
-[Web Components authentication]:19_webcomponents_authentication.md#programmatic-authentication
-[the same events as the Dashboard component]:https://sdk.gooddata.com/gooddata-ui-apidocs/docs/sdk-ui-dashboard.dashboardeventtype.html
+[The full list of available localizations]:https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui/src/base/localization/Locale.ts
+[The Web Components authentication]:19_webcomponents_authentication.md#programmatic-authentication
```
Comment (andriichumak) [inline docs/19_webcomponents_dashboard.md:54]: I think this link will get broken, as `The` part is added here and not in the article body.

The link text in the body and here must be the same. I'll fix it in a moment.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1197/ : SUCCESS in 1m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/800/ : SUCCESS in 6s

File: docs/19_webcomponents_authentication.md [hunk 2]
```diff
@@ -26,7 +23,7 @@ Meanwhile, all custom elements present on the page will render a loading animati
 
 If you want to customize the authentication flow, you'll need to provide the authenticated backend by yourself.
 Web Components library hosts several more files that you can import to your browser runtime and that expose
-backend factories for GoodData.CN, GoodData Cloud and platform.
+a backend factory for GoodData.CN and GoodData Cloud (platform code name `tiger`).
```
Comment (no23reason) [inline docs/19_webcomponents_authentication.md:26]: Nitpick: I would drop the "platform" work, just codename tiger

File: docs/19_webcomponents_authentication.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+If you need more control over the authentication flow, you can also [authenticate the user programmatically](#programmatic-authentication).
+
+Currently, the only valid value for the `auth` parameter is `sso`. When set, the script will check the authentication
+status and, if needed, the script will redirect the user to the SSO login page. See more about [`?auth=sso` below](#automatic-authentication).
+
+If the `auth` parameter is not provided or has a value other than `sso`, the library will not
+run the auto-authentication and will expect you to run the [programmatic authentication](#programmatic-authentication).
+Meanwhile, all custom elements present on the page will render a loading animation.
+
+> All your users must have a GoodData account and have access to your GoodData workspace. Ifs users do not have a GoodData account or do not have access to the workspace, see [Grant users access to your workspace][1].
```
Comment (no23reason) [inline docs/19_webcomponents_authentication.md:20]: Typo: Ifs

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1203/ : SUCCESS in 2m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/805/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1204/ : SUCCESS in 1m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/806/ : SUCCESS in 8s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR541

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1205/ : SUCCESS in 1m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/165/ : SUCCESS in 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/360/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul/2/ : FAILURE in 3m 00s (non-voting)

**Pull request merged**